doc/source/tune/_tutorials/tune-pytorch-cifar.rst

```diff
@@ -0,0 +1,212 @@
+.. _tune-pytorch-cifar:
+
+How to use Tune with PyTorch
+============================
+
+Hyperparameter tuning can make the difference between an average model and a highly
+accurate one. Often simple things like choosing a different learning rate or changing
+a network layer size can have a dramatic impact on your model performance. Fortunately,
+Tune makes exploring these optimal parameter combinations easy - and works nicely
+together with PyTorch.
+
+In this walkthrough, we will show you how to integrate Tune into your PyTorch
+training workflow. We will follow `this tutorial from the PyTorch documentation
+<https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html>`_ for training
+a CIFAR10 image classifier.
```

Comment:
BTW, now reading this, I think we will want this to be the first paragraph (and percolate relevant changes).